services/community-site/src/pages/Profile/ViewProfile.tsx

```diff
@@ -101,7 +100,7 @@ function ViewProfileDetails({ points, openEditProfile, openKYCModal }: ViewProfi
   useEffect(() => {
```

Comment:
Maybe here instead of calling it with the old account, we could introduce a new useEffect.
```suggestion
  useEffect(() => {
  if(checksumAccount)
  ...
  }, [checksumAccount])
```
And then use the chsckAccount as dep